### PR TITLE
Explicitly mark modules as Safe

### PR DESCRIPTION
**Pull request by phadej**
Part of https://github.com/ekmett/lens/issues/959 effort

### CODE REVIEW

File: src/Data/Bifunctor/Biap.hs [hunk 1]
```diff
@@ -10,6 +10,10 @@
 {-# LANGUAGE DeriveGeneric              #-}
 #endif
 
+-- This module uses GND
+#if __GLASGOW_HASKELL__ >= 702
```
Comment (RyanGlScott) [inline src/Data/Bifunctor/Biap.hs:14]: This could be combined with the existing `#if __GLASGOW_HASKELL__ >= 702 ... #endif` above.

Comment (phadej) [reply src/Data/Bifunctor/Biap.hs:14]: I'd rather keep them separate. *If* GND eventually becomes Safe :)

**Comment by phadej** [comment]
I kindly ask for release of `bifunctors`, so I can continue with `semigroupoids` and `profunctors`.

**Comment by RyanGlScott** [comment]
I've uploaded [`bifunctors-5.5.9`](http://hackage.haskell.org/package/bifunctors-5.5.9) to Hackage with these changes.

**Pull request merged**